package ru.bortnik.second.exception;

public class StorageException extends RuntimeException {

    public StorageException(Throwable cause) {
        super(cause);
    }
}
